centreon/www/widgets/hostgroup-monitoring/src/export.php

```diff
@@ -75,47 +75,58 @@
 $hostStateLabels = [0 => "Up", 1 => "Down", 2 => "Unreachable", 4 => "Pending"];
 
 $serviceStateLabels = [0 => "Ok", 1 => "Warning", 2 => "Critical", 3 => "Unknown", 4 => "Pending"];
+try {
+    $query = <<<'SQL_WRAP'
+        SELECT SQL_CALC_FOUND_ROWS DISTINCT
+            1 AS REALTIME,
+            name
+        FROM hostgroups 
+        SQL_WRAP;
+
+    if (isset($preferences['hg_name_search']) && $preferences['hg_name_search'] != "") {
+        $tab = explode(" ", $preferences['hg_name_search']);
+        $op = $tab[0];
+        if (isset($tab[1])) {
+            $search = $tab[1];
+        }
+        if ($op && isset($search) && $search != "") {
+            $query = CentreonUtils::conditionBuilder(
+                $query,
+                "name " . CentreonUtils::operandToMysqlFormat($op) . " '" . $dbb->escape($search) . "' "
```

Comment:
CentreonDB::escape is deprecated, you must to use CentreonDB::escapeString now ;)

Reply:
ok